microsoft-edge/devtools-guide-chromium/overview.md

```diff
@@ -129,6 +129,8 @@ To have DevTools automatically open whenever you open a new tab in the browser:
    microsoft-edge-dev --auto-open-devtools-for-tabs
    ```
    
+> [!IMPORTANT]
+> There must be no running Edge processes when launching via `auto-open-devtools-for-tabs`. You may need to disable Startup Boost in `edge://settings/system`. More information about Startup Boost can be found here: [Get help with startup boose - Microsoft Support](https://support.microsoft.com/topic/get-help-with-startup-boost-ebef73ed-5c72-462f-8726-512782c5e442).
 
```

Comment:
Should some of this wording be added from PR main page, for more clarification?

auto-open-devtools-for-tabs requires there to be no running Edge processes. 

Most users have the Startup Boost feature enabled, which keeps a minimal Edge process running in the background. 

This interferes with the auto-open-devtools-for-tabs feature and would need to be disabled to use it.

Or users would have to task-kill the Startup Boost Edge process before running the auto-open-devtools-for-tabs command.